pkg/generators/snippets/deepcopy_test.go

```diff
@@ -23,12 +23,6 @@ func TestGenerateDeepCopyInto(t *testing.T) {
 		wantErr error
 		want    string
 	}{
-		{
-			// nil pointer ctx
-			ctx:     nil,
-			t:       newTestNamespaceType(),
-			wantErr: errors.New("nil pointer"),
-		},
```

Comment:
nil pointer tests for ctx are not needed anymore for `Generate*` functions. This check is now responsibility of `writeSnippetWithArgs`